sherpa/utils/__init__.py

```diff
@@ -1633,7 +1642,7 @@ def run_tasks(procs, err_q, out_q, num):
     # return list(numpy.concatenate(results))
     # Remove extra dimension added by split
     vals = []
-    [ vals.extend(result) for result in results ]
+    [vals.extend(result) for result in results]
```

Comment:
I've been tempted a few times to rewrite this line as it is not particularly clear/pythonic!